froi/algorithm/array2qimage.py

```diff
@@ -365,3 +368,13 @@ def null_image(h, w):
     new_array = _np.zeros((h, w, 4), dtype=_np.uint8)
     new_array[..., 3] = 255
     return new_array
+
+
+if __name__ == '__main__':
```

Comment:
This part can be removed.
Usually, the code for tests should  be saved in the `tests` directory.